### ISSUE TITLE
also fetch events from graphite

### ISSUE THREAD

**Issue by Finkregh** [vimeo/timeserieswidget#7]
it would be very nice if one could also use the events that graphite uses (as e.g. http://kenhub.github.io/giraffe/ does)

**Comment by Dieterbe** [comment]
yeah, i would welcome the feature, but i'm not interested in it myself

**Comment by Finkregh** [comment]
might a bounty change that? :)

**Comment by Dieterbe** [comment]
maybe..
